bin/tictactoe

```diff
@@ -1,3 +1,8 @@
 #!/usr/bin/env ruby
 
```

Comment:
Hi, Cernan (@cernanb) . I made some changes in the code, as per your guidance. Please let me know in case any further improvements might be needed.
Thank you.